xfel/merging/application/phil/phil.py

```diff
@@ -129,14 +129,18 @@
   .help = The filter section defines criteria to accept or reject whole experiments
   .help = or to modify the entire experiment by a reindexing operator
   .help = refer to the select section for filtering of individual reflections
+  .help = only unit_cell, resolution, and n_obs are implemented
   {
   algorithm = n_obs reindex resolution unit_cell report
-    .type = choice
+    .type = strings
     .multiple = True
```

Comment:
If .type = choice and .multiple = true, then params.filter.algorithm should be a list of already validated strings.  Is there a reason that doesn't work?